scripts/genvp.py

```diff
@@ -1453,6 +1453,8 @@ def parseStructInfo(self, xml):
                         elif tail == '[' and enum != None and enum.tail == ']':
                             # [<enum>] case
                             structDef.members[name].arraySize = enum.text
+                            if structDef.members[name].arraySize in self.constants:
+                                structDef.members[name].arraySize = int(self.constants[structDef.members[name].arraySize])
```

Comment:
This is not how this was originally meant to be handled. There's a method in `VulkanRegistry` called `evalArraySize` which is used to map the constant values to their actual integer value.

Probably some call to `evalArraySize` is missing somewhere if this caused issues.
I'll try it on my end what goes wrong with a pipelineCacheUUID.

Reply:
Please find the suggested fix in PR #158.

I'm not saying that this PR does not fix the same issue, but it does introduce some inconsistency in array size handling.
The original bug was simply a result of me forgetting to update existing code when adding support for dynamic array size evaluation.

Reply:
I'd rather take your PR since its more idiomatic anyhow.
I just did this so I could verify that the profile vulkaninfo created would successfully run through the gen script (it does, with this fix in place).